filters.json.example

```diff
@@ -426,11 +426,13 @@
     },
     "eggs":{
         "enabled":"True",
+        "gymname_contains":["sprint","level badge","phone skin","starbucks"],
```

Comment:
Filter examples need to be updated with the change in verbiage and to match the actual checks.

Reply:
'gymname_contains' is the verbiage.  can you clarify what you are asking?